ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -94,24 +73,10 @@ tests:
         NUM_NODES=4 \
           make e2e-tests
       dependencies:
-      - env: CREATE_VM_IMAGE
-        name: tekton-task-create-vm
-      - env: COPY_TEMPLATE_IMAGE
-        name: tekton-task-copy-template
-      - env: EXECUTE_IN_VM_IMAGE
-        name: tekton-task-execute-in-vm
-      - env: GENERATE_SSH_KEYS_IMAGE
-        name: tekton-task-generate-ssh-keys
-      - env: DISK_VIRT_CUSTOMIZE_IMAGE
-        name: tekton-task-disk-virt-customize
-      - env: DISK_VIRT_SYSPREP_IMAGE
-        name: tekton-task-disk-virt-sysprep
-      - env: WAIT_FOR_VMI_STATUS_IMAGE
-        name: tekton-task-wait-for-vmi-status
-      - env: MODIFY_VM_TEMPLATE_IMAGE
-        name: tekton-task-modify-vm-template
-      - env: MODIFY_DATA_OBJECT_IMAGE
-        name: tekton-task-modify-data-object
+      - env: TEKTON_TASKS_IMAGE
+        name: tekton-tasks
+      - env: DISK_VIRT_IMAGE
```

Comment:
Rename it to `TEKTON_TASK_DISK_VIRT_IMAGE` for consistency?

Reply:
done